deployments/cluster/witness-load-balancer.nix

```diff
@@ -30,12 +52,44 @@ domain: { config, pkgs, ... }:
       '';
     };
 
-    virtualHosts."${config.deployment.route53.hostName}" = {
-      enableACME = true;
+    upstreams.educator = {
+      servers."educator:4040" = {};
+      extraConfig = "keepalive 32;";
+    };
+
+    upstreams.faucet = {
+      servers."127.0.0.1:4014" = {};
+      extraConfig = "keepalive 32;";
+    };
+
+    virtualHosts= {
+      "${uris.witness}".locations."/".proxyPass = "http://witness";
+      "${uris.educator}".locations."/".proxyPass = "http://educator";
+      "${uris.explorer}".locations."/".root = pkgs.disciplina-explorer-frontend.override { witnessUrl = "//${uris.witness}"; };
```

Comment:
If you add a proxypass on this vhost, there will be no need to rebuild the explorer